nomad/server.go

```diff
@@ -612,6 +615,18 @@ func (s *Server) State() *state.StateStore {
 	return s.fsm.State()
 }
 
+// Regions returns the known regions in the cluster.
+func (s *Server) Regions() []string {
+	s.peerLock.RLock()
+	defer s.peerLock.RUnlock()
+
+	regions := make([]string, 0, len(s.peers))
```

Comment:
What if I have multiple servers per region. You probably want to make a map to keep track of unique regions